docs/api-guide/fields.md

```diff
@@ -366,6 +366,7 @@ Used by `ModelSerializer` to automatically generate fields if the corresponding
 - `allow_blank` - If set to `True` then the empty string should be considered a valid value. If set to `False` then the empty string is considered invalid and will raise a validation error. Defaults to `False`.
 - `html_cutoff` - If set this will be the maximum number of choices that will be displayed by a HTML select drop down. Can be used to ensure that automatically generated ChoiceFields with very large possible selections do not prevent a template from rendering. Defaults to `None`.
 - `html_cutoff_text` - If set this will display a textual indicator if the maximum number of items have been cutoff in an HTML select drop down. Defaults to `"More than {count} items…"`
+- `use_value` - If set to `True` then input and output values changed with choice field values.
```

Comment:
Anyone got thoughts on how we could rephrase this to be more clear. I know from the discussion what the change is, but I wouldn't be able to understand it from this description.